template_init_data.json

```diff
@@ -0,0 +1,23 @@
ect ([Here is the generic version of that URL](https://console.developers.google.com/apis/credentials/oauthclient/)). \n\n\n\n**REACT_APP_AUTH_NAME**\n\nName that will be displayed on the admin login button. For example, if you set `REACT_APP_AUTH_NAME` to \"My Project\", the text would display as\n\n![](https://raw.githubusercontent.com/binaris/dev-docs-content/master/assets/auth-button-example.png)\n\n\n### Adding Content\n\nTo add content on the site, navigate to `https://<base-url>/editor`. At this point, you'll be asked to authenticate using your previously configured Google OAuth credentials. After successfully authenticating, you'll be redirected to the editor page.\n\n![](https://raw.githubusercontent.com/binaris/dev-docs-content/master/assets/dev-editor-screen.png)\n\n_The editor page_\n\nThe bottom half of the page can be used to write markdown, which will be parsed and displayed in real time. The \"Update\" button will update the content that you're currently editing (or create it if it doesn't exist). \"Select a Post\" allows you to retrieve existing posts and edit them as desired.\n\nFor a post to be recognized and displayed by the site, it must include a frontmatter with the following keys (additional keys are ok, these ones are required):\n\n```yaml\n---\ntitle:\nroute:\ncategory:\n---\n```\n\n**Title** controls the displayed name of the content on your site. \n\n**Route** determines what sub-URL the content will be available at. Valid routes can only contain lowercase letters, numbers and dashes (no spaces). For example, if my route was 'example-content', the content would be available at `https://<base-url>/example-content`.\n\n**Category** controls the sidebar category that your content will be displayed in. Note that this isn't strictly required, although your content will be completely hidden if omitted. Currently, the valid categories are:\n\n* Getting Started\n* Main Concepts\n* Other Goodies\n\n\n> When posts are created, they are initially disabled by default. You'll need to visit the `/admin` page to enable your post.\n\n### Managing Content\n\nTo manage content on the site, navigate to `https://<base-url>/admin`. Similarly to the `/editor` page, you'll be asked to authenticate using your previously configured Google OAuth credentials. After successfully authenticating, you'll be redirected to the `/admin` page.\n\n![](https://raw.githubusercontent.com/binaris/dev-docs-content/master/assets/dev-admin-screen.png)\n\n_The admin page_\n\nThe admin page provides mechanisms to enable, disable and delete pages. A home page for the site can also be set, using the \"Set as home\" button.\n\n> Note: only a single page can be set as the home route\n\n\nWe love feedback! To provide feedback about this template (or any other part of the Reshuffle experience), please reach out on [Twitter](https://twitter.com/reshufflehq), [Discord](https://discord.gg/M8CC5hy) or open an issue on [Github](https://github.com/reshufflehq).\n\n\n<br />",
```

Comment:
Nit: `[follow the steps described in this Google documentation.](https://developers.google.com/identity/protocols/OAuth2).` ends up having two full stops

Also Nit: Perhaps change "if omitted" to "if it is omitted" ?